bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -350,6 +350,20 @@ private function addRoutingNode(ArrayNodeDefinition $rootNode)
                 ->arrayNode('routing')
                     ->addDefaultsIfNotSet()
                     ->children()
+                        ->arrayNode('defaults')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->scalarNode('bundle')
+                                    ->defaultValue(PIMCORE_SYMFONY_DEFAULT_BUNDLE)
```

Comment:
Imho we can completely get rid of those constants and define the default values in the config-tree definition. 
If someone wants to use constants anyway, there's a special syntax for the use in the config.

Reply:
I removed the constants and add an BC notice